app/views/index.html

```diff
@@ -46,6 +46,7 @@
             window.__ROOM_NAME__ = "${roomName}";
             window.__USER_ADMIN__ = "${user.isAdmin()}";
             window.__USER_GUEST__ = "${user.isGuest()}";
+            document.title="${room_name"} + " : breaker";
```

Comment:
n.b. the document title _should_ be getting set via [this component](https://github.com/larvalabs/breaker/blob/master/web/document/DocumentTitle.jsx). Were you not seeing it set?